CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 0.10.0
+
+* `nerves_hub_link_common 0.2.0`
+
+* Enhancements
+  * Report `device_api_version` in web connections
+  * Report `currently_downloading_uuid` when joining channel to aid
+    in resumable downloads
+  * Optional `NervesHubLink.Client.reboot/0` callback behavior
+
+* Fixes
+  * Check `firmware_url` is valid before attempting update
+  * set default `fwup_devpath` in the config
```

Comment:
I think I may have missed reviewing the `fwup_devpath` or didn't appreciate it. Setting a default is probably not a feature, since if it ever got used, it would mask a really serious issue.